README.md

```diff
@@ -44,6 +55,17 @@ Usage:
 dbt run-operation fivetran_utils.generate_columns_macro --args '{"table_name": "promoted_tweet_report", "schema_name": "twitter_ads", "database_name": "dbt-package-testing"}'
 ```
 ----
+#### median ([source](macros/median.sql))
+This macro is used to return the median value of a field with cross db functionality. The default median calulcation within the macro is the `median` function. The BigQuery median calculation within the macro is the `percentile_cont` function.
+
+**Usage:**
+```sql
+{{ fivetran_utils.median(median_field='time_to_close', partition_field='id', percent=0.5) }}
+```
+**Args:**
+* `median_field` (required): Name of the field of which you are determining the median value.
+* `partition_field` (required): Name of the field you want to partition by to determine the median value.
+* `percent` (default = optional, bigquery = required): The percent necessary for `percentile_cont` to determine the median value. By defualt the value is `0.5`. 
```

Comment:
is there a case in which someone would use a different percent to calculate the median? if not, might be simpler to remove this argument and just always use 0.5 🤷

Reply:
Completely changed this macro to instead be a percentile macro which we can use for median if we want.